pydantic_extra_types/color.py

```diff
@@ -7,18 +7,20 @@
 In these cases the _last_ color when sorted alphabetically takes preferences,
 eg. `Color((0, 255, 255)).as_named() == 'cyan'` because "cyan" comes after "aqua".
 """
+from __future__ import annotations
```

Comment:
Most of the changes here are becuase of this.

Reply:
👍🏻